ports/superdungeonmuncher/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "superdungeonmuncher.zip",
+  "items": [
+    "Super Dungeon Muncher.sh",
+    "superdungeonmuncher"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Super Dungeon Muncher",
+    "porter": [
+      "turbo_16bit"
+    ],
+    "desc": "Super Dungeon Muncher combines puzzles and rogue-like action with a unique twist. The Muncher chases you in real time as you progress through the world. Act fast and think quick or the Muncher will get you.",
+    "desc_md": null,
+    "inst": "Download the game from https://n-nexy.itch.io/super-dungeon-muncher and extract it. Put the data.win from the extracted folder into your ports/superdungeonmuncher folder. ",
```

Comment:
the inst_md field is for instructions with markdown.
You can put the text it bit more fancy for the website. Like links etc.

The inst and desc field are for use in the PortMater GUI on device.
So space is limited

Reply:
fancied it up a bit :)